tests/Agent/IntegrationTests/IntegrationTests/Owin/OwinWebApiAsyncTests.cs

```diff
@@ -81,15 +81,15 @@ public void Test()
 
             var generalMetrics = new List<Assertions.ExpectedMetric>
             {
-                new Assertions.ExpectedMetric { metricName = @"Supportability/AnalyticsEvents/TotalEventsSeen", callCount = ExpectedTransactionCount },
+                new Assertions.ExpectedMetric { metricName = @"Supportability/AnalyticsEvents/TotalEventsSeen", CallCountAllHarvests = ExpectedTransactionCount },
```

Comment:
These are sometimes split across two harvest cycles, so we need to take the total count